meson.build

```diff
@@ -219,12 +219,12 @@ endif
 libjcat = dependency('jcat', version : '>= 0.1.0', fallback : ['libjcat', 'libjcat_dep'])
 libjsonglib = dependency('json-glib-1.0', version : '>= 1.1.1')
 valgrind = dependency('valgrind', required: false)
-libcurl = dependency('libcurl', version : '>= 7.47.0')
-if libcurl.version().version_compare('>= 7.56.0')
-  conf.set('HAVE_LIBCURL_7_56_0', '1')
-endif
-if libcurl.version().version_compare('>= 7.62.0')
-  conf.set('HAVE_LIBCURL_7_62_0', '1')
+if get_option('curl')
+  libcurl = dependency('libcurl', version : '>= 7.56.0')
```

Comment:
I guess the reason you let it go back to 7.47.0 was just for the version on the fuzzer right?  So that's why bumping back up to 7.56.0 is OK?

Reply:
Exactly that. The static binaries were huuuge with all the libcurl deps linked in, and "no libcurl" is better than "old libcurl".